src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -30,6 +77,10 @@ class PaymentInformationEditModal extends React.Component {
     responseError: PropTypes.object,
   };
 
+  state = {
```

Comment:
No need to use Redux like I did with the new Profile forms as those had a much more complex component structure. In this case everything is contained in a single component.